extensions/elytron-security/deployment/src/test/java/io/quarkus/security/test/jdbc/RolesEndpointClassLevel.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.security.test.jdbc;
+
+import javax.annotation.security.RolesAllowed;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.SecurityContext;
+
+/**
+ * Test JAXRS endpoint with RolesAllowed specified at the class level
+ */
+@Path("/roles-class")
+@RolesAllowed("user")
+public class RolesEndpointClassLevel {
```

Comment:
Why don't you reuse the existing `io.quarkus.security.test.RolesEndpointClassLevel` ?